package.nls.ja.json

```diff
@@ -0,0 +1,55 @@
+{
+  "azure-iot-toolkit.name": "Azure IoT Hub",
```

Comment:
For no translation part, we could remove it from package.nls.ja.json. It would fall back to package.nls.json